scripts/ci/coverage-gate.sh

```
@@ -0,0 +1,255 @@
+${project_rows}
+---
+*Generated by coverage-gate.sh*
+EOF
+}
+
+# Main entry point (only runs when script is executed directly, not sourced)
+main() {
+  local coverage_file="./coverage-merged/Cobertura.xml"
+  # Defaults align with org policy; override via env vars if needed
+  local threshold="${COVERAGE_THRESHOLD:-90}"
+  local branch_threshold="${BRANCH_COVERAGE_THRESHOLD:-70}"
```

Shellcheck (0.11.0)

[warning] branch_threshold appears unused. Verify use (or export if used externally).

(SC2034)